CRM/Donrec/Exporters/EmailPDF.php

```diff
@@ -140,6 +140,21 @@ protected function sendEmail($receipt, $pdf_file, $is_test, $snapshot_line_id) {
       // load contact data
       $contact = civicrm_api3('Contact', 'getsingle', array('id' => $receipt['contact_id']));
 
+      if (CRM_Donrec_Logic_Settings::get('email_location_type_id')) {
+        // load email address from the configured location type
+        $email_location_type_id = CRM_Donrec_Logic_Settings::get('email_location_type_id');
```

Comment:
Let's avoid calling `CRM_Donrec_Logic_Settings::get()` twice.

Reply:
It's done.